src/ProofManagerV1.sol

```diff
@@ -42,6 +45,11 @@ contract ProofManagerV1 is
     ///     Minimizes the proving downtime in case of communication failure.
     uint256 private constant ACK_TIMEOUT = 2 minutes;
 
+    /// @dev Hard-coded constant on maximum timeout after.
+    ///     Proving Networks have max 3 hours(can be less) to submit a proof once committed.
```

Comment:
Maybe `Proving Networks have min(MAX_TIMEOUT_AFTER, proving_request.timeout) time to prove the request. This guarantees that there will be no proofs accepted after 3h has passed since submission.`